src/console_cmds.cpp

```diff
@@ -1444,8 +1499,12 @@ static bool ConStartAI([[maybe_unused]] uint8_t argc, [[maybe_unused]] char *arg
 				size_t name_length = e - argv[1];
 				e++;
 
-				int version = atoi(e);
-				config->Change(std::string(argv[1], name_length), version, true);
+				auto version = ParseInteger(e, 10);
```

Comment:
Technically, there can be negative versions of AIs. Is `ParseInteger` able to read something like `-123`

Reply:
No, negative values are not valid.

Reply:
Technically, yes you can add a negative version for a script.

Though good luck force-selecting version -1. Or actually getting the latest version when that's -1.

So negative version numbers are just stupid, and as such I'm not supporting them here. If you really need to, just misuse characteristics of casting an unsigned integer into a signed one.